client/js/views/Home.vue

```diff
@@ -24,8 +30,8 @@
               </div>
             </div>
           </v-col>
-          <v-col id="side-nav" cols="3">
-            <img id="logo" alt="Vue logo" src="../img/logo.png" />
+          <v-col id="side-nav" :cols="$vuetify.breakpoint.mdAndDown ? 12 : 3">
+            <img v-if="!$vuetify.breakpoint.mdAndDown" id="logo" alt="Vue logo" src="../img/logo.png" />
```

Comment:
Same here with `lgAndUp`

Reply:
So if the first one is changed. We would have to change the number of cols or the image explodes in size. The second in this image where we are checking if it is not mdAndDown I have changed.